web/src/components/snapshots/SnapshotSchedule.jsx

```diff
@@ -587,10 +593,11 @@ class SnapshotSchedule extends Component {
                 </p>
                 <p className="u-fontSize--small u-textColor--bodyCopy u-fontWeight--normal u-lineHeight--normal u-marginBottom--10">
                   The Admin Console can reclaim space by automatically deleting
-                  older scheduled snapshots.
+                  older scheduled {featureName}s.
                 </p>
                 <p className="u-fontSize--small u-textColor--bodyCopy u-fontWeight--normal u-lineHeight--normal u-marginBottom--10">
-                  Snapshots older than this will be deleted.
+                  {Utilities.toTitleCase(featureName)}s older than this will be
+                  deleted.
```

Comment:
I think we could condense this all into something like this. WDYT?
>Choose how long to retain backups before they are automatically deleted.

Reply:
I like it. Simpler and indicates the TTL is applied to _all_ backups.